### PR TITLE
Release 1.4.0

### PR DESCRIPTION
**Pull request by aaronjudd**
Release 1.4.0

##  Changes / Fixes

- __Meteor 1.5.1__
- __Adds inline validation to variant &amp; child variant form (#2531)__
- __Longer timeout for circleci to run build.sh (#2565)__
- __Translations updated based on development (#2566)__
- __Fix You may not observe cursor with fields id: 0 (#2555)__
- __Fix LoadMoreProducts button not working as it should (#2554)__
- __Provide sort parameter to Products subscription and remove duplicate sorting of products (#2552)__
- __Fix issue 2535. Transliteration in Safari/IE (#2553)__
- __Fix 2547: vertically-center navbar link using @navbar-height height variable instead of percentage height (#2551)__
- __Update eslint to v4 (#2542)__
- __Fix order action button reverting status to processing (#2541)__
- __Update Docker base to v2.0.2 (#2537)__
- __Fix Exception in delivering result of invoking &#39;email/verifySettings&#39; (#2512)__
- __Fix unable to open fulfill order action view (#2530)__
- __Fix for issue #1993 (#2518)__
- __Fix for no scrollbar in Dashboard (#2511)__
- __Convert product grid to React: Milestone 57 (#2402)__
- __Fix Errors importing exported product data  (#2513)__
- __Fix send sms method to read settings of shop and buyer userId (#2508)__
- __Fix issue dragging images in PDP when settings drawer is open (#2506)__
- __Add Tunisian Dinar currency (#2517)__
- __Default to using shop country code or GeoIP result for new addresses (#2515)__
- __Fix missing reference (#2507)__
- __Show proper error messages when Stripe payments fail (#2499)__
- __Fix linting problems introduced in new SortableTable (#2505)__
- __New SortableTable component to replace MeteorGriddle (#2416)__
- __Fix ESLint4 errors correct eslint config errors (#2450)__
- __Remove redundant risul:moment-timezone (#2491)__
- __Fix hard refresh on clicking tag link on product detail page (#2489)__
- __Fixed product images not displaying on order completed page (#2490)__
- __Updated timeout time. (#2492)__
- __Fix error when inviting already invited user (#2433)__
- __Rename emailTableColumn -> shippoTableColumn (#2564)__
- __LingoHub based on development (#2566)__
- __Longer timeout for circleci to run build.sh (#2565)__
- __Adds inline validation to variant & child variant form (#2531)__

## Thanks To Our Community Contributors
**Changes and Fixes**
@prinzdezibel 
@nbaldwin
@stoufa88
@lcampanis  

**Critical Issues Reported**
@jaden-chen
@stevaras

### CODE REVIEW

**Comment by zenweasel** [comment]
I'm getting an error when I search for an order and click on it. It doesn't always happen though.
```
Exception in template helper: TypeError: Cannot read property '_id' of undefined
    at Object.order (http://localhost:3000/app/app.js?hash=cb13563f06c11b695487f9efdd0e341bfc5994c6:47598:26)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3051:16
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1715:16
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3103:66
    at Function.Template._withTemplateInstanceFunc (http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3744:12)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:3102:27
    at Object.Spacebars.call (http://localhost:3000/packages/spacebars.js?hash=ebf9381e7fc625d41acb0df14995b7614360858a:172:18)
    at http://localhost:3000/app/app.js?hash=cb13563f06c11b695487f9efdd0e341bfc5994c6:45892:22
    at Blaze.View.<anonymous> (http://localhost:3000/packages/spacebars.js?hash=ebf9381e7fc625d41acb0df14995b7614360858a:264:18)
    at http://localhost:3000/packages/blaze.js?hash=f33d3dfed63a491d24e3aa07ad66c24b5fe8c761:1934:20
```

**Comment by zenweasel** [comment]
Error in console browsing in Safari private mode. Will double-back to check but I suspect that this happens in `master` as well.

dashboard_orders

**Comment by zenweasel** [comment]
Filter by tag no longer works in the product search

**Comment by aaronjudd** [comment]
The Safari error is a result of trying to use localStorage (appears to be CFS) in Safari while in Private mode (localStorage is not permitted).  I would have thought we'd have seen this long before though.

**Comment by spencern** [comment]
The Safari error has certainly been seen before. I thought #2518 might have addressed this in some part. I feel like this is a bug we've seen and confirmed fixed a few times at least

**Comment by spencern** [comment]
I think #2089 also fixed similar symptoms, but perhaps we just solved the issue that was causing Reaction to not load. Do we need to suppress LocalStorage related warnings in Safari?

**Comment by prinzdezibel** [comment]
@spencern #2015 doesn't address the warnings in general. It just provides a mechanism to re-login when coming from an external site without using localStorage at all.

File: imports/plugins/included/shippo/client/settings/shippoTableColumn.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Icon } from "/imports/plugins/core/ui/client/components";
+
+class EmailTableColumn extends Component {
```
Comment (kieckhafer) [inline imports/plugins/included/shippo/client/settings/shippoTableColumn.js]: Yes, that's exactly why.

**Pull request merged**